client/components/gpu-listings/GpuCard.js

```diff
@@ -0,0 +1,37 @@
+import style from './GpuCard.module.css'
+import Image from 'next/image'
```

Comment:
It's typically a common practice to sort imports in an order that is visually easier to scan, for example

<npm packages>

<your component imports>
<your icon / img imports>
<your util function imports>

<your style imports>

Reply:
Ok sounds good, I will update and do that moving forward.